src/sql/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -49,3 +81,24 @@ CommandsRegistry.registerCommand({
 		}
 	}
 });
+export class ADSWebLite implements IWorkbenchContribution {
+	constructor(
+		@ICommandService private commandService: ICommandService,
+	) {
+		this.registerEditorOverride();
+	}
+
+	private async registerEditorOverride(): Promise<void> {
+		if (product.quality === 'tsgops-image') {
+			await this.commandService.executeCommand('workbench.extensions.action.hideSettings');
+			await this.commandService.executeCommand('workbench.extensions.action.hidePanel');
+			let array = ['workbench.view.search', 'workbench.view.explorer', 'workbench.view.scm', 'workbench.view.extensions'];
+			for (let j = 0; j < array.length; j++) {
+				await this.commandService.executeCommand('workbench.extensions.action.hideActivityBarViewContainers', array[j]);
+			}
+		}
+	}
+}
+
+Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench)
```

Comment:
This should only be registered when the quality is tsgops (and you can then remove the product.quality check in the contribution)